locale/zh-CN/How-to-Setup.md

```diff
@@ -36,6 +36,7 @@ has_children: true
 3. 双击安装 appx 文件。
 
 ## 完成安装后的操作
+### [若你使用 WSL1 ，你将**必须**修改一下 glibc 包。更多详情，请查看已知问题章节。](Known-issues.md#systemdsystemctl)
```

Comment:
Sorry, please link to Known-issues.md#wsl1--wsl2

Reply:
> Sorry, please link to Known-issues.md#wsl1--wsl2

Oh. Wait..